www/ifdb.css

```diff
@@ -774,13 +765,23 @@ div.searchHelpList td {
 Tabs for browse-mode search types
 */
 div.browseBar {
-    position: relative;
-    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    width: calc(100% + 2rem);
     font-size: 85%;
     background: #e0e0e0;
+    margin-bottom: 1em;
+    margin-left: -1rem;
+    padding-left: 1rem;
+    text-size-adjust: none;
+    -webkit-text-size-adjust: none;
+    flex-wrap: wrap;
 }
 div.browseTabs {
-    padding: 6px 0 0 4px;
+    padding: 6px 1em 0 4px;
+    overflow: scroll;
```

Comment:
Looks like this is causing a partial white frame in my Mac Chrome browser:

![Screen Shot 2021-04-06 at 7 23 46 AM](https://user-images.githubusercontent.com/1774931/113728641-1d4b9a00-96ab-11eb-852a-2558cf97752e.png)

Reply:
Everything else looks good. I tested in Chrome with the device emulator. Also, checked a couple of the improvements at the dev site on on my iPhone.

Reply:
I removed that bit in the latest version of the PR. https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/213 is no longer fixed in this PR.